testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/standard/trino-init-hive3.sh

```diff
@@ -3,7 +3,7 @@
 set -xeuo pipefail
 
 for hive_properties in $(find '/docker/presto-product-tests/conf/presto/etc/catalog' -name '*hive*.properties'); do
-    echo "Updating $hive_properties for HDP3"
+    echo "Updating $hive_properties for Hive3"
```

Comment:
Changes here (file rename, comment change) are no longer related to the actual commit, but still good thing to do.
Let's make them into separate commit (or PR).